src/Components/TrainingListPage.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../Store/configureStore';
+import fixtures from '../Fixtures/TestFixtures';
+
+import ConnectedTrainingListPage, { TrainingListPage } from './TrainingListPage';
+
+const storeInstance = store;
```

Comment:
You can remove this line and use `store` directly on the test below

(That useless line is from me and I removed it from the other existing test in this PR #47 )